sites/next.skeleton.dev/src/examples/components/progress-rings/Example.svelte

```diff
@@ -0,0 +1,11 @@
+<script lang="ts">
+	import { ProgressRing } from '@skeletonlabs/skeleton-svelte';
+
+	let value = $state(25);
+	let max = 100;
+</script>
+
+<div class="grid grid-cols-1 gap-4">
+	<ProgressRing {value} {max} />
```

Comment:
```suggestion
	<ProgressRing value={value} max={max} />
```
in v2 we had the issue that some new users are not understanding this syntax, so maybe we want to stick to the same one for all examples.

Reply:
Per this, I think we opt for the condensed version. It reads better IMO. If folks have issues, we'll add some onboarding documentation around this convention.